xapian-applications/omega/Makefile.am

```diff
@@ -44,7 +44,7 @@ BUILT_SOURCES = extra/omegascript.vim my-html-tok.h mimemap.h namedents.h
 
 MAINTAINERCLEANFILES = my-html-tok.h mimemap.h mimemap.stamp namedents.h
 
-EXTRA_PROGRAMS = omindex_pdf omindex_ebook
+EXTRA_PROGRAMS = omindex_pdf omindex_ebook omindex_idocs
```

Comment:
I think we want to name these helper binaries after the **library** not the **format** (I'm fairly sure I mentioned this re `omindex_pdf` which I think would be better named `omindex_poppler`, but it seems that never got addressed).

If we name the helper after the format it's harder to cleanly support two helpers which can handle the same format.  E.g. `omindex_wv2` and `omindex_libreofficekit` rather than `omindex_msword` and being stuck for a good way to name the other one or clearly tell them apart.  We really want these helpers to have a stable name so that we can provide a way for users to configure them at run time, and the most stable name is the name of the library they use.

This also works better for handlers which handle more than one format, which is quite common.

Reply:
Yes, I remember that. Actually, it seems that I misunderstand that and only change the name of the file (handler_pdf.cc -> handler_poppler.cc).
I am going to fix it.